setup.py

```diff
@@ -8,6 +9,18 @@
     "kazoo==2.0",
 ]
 
+# Regex matching pattern for v followed by 2 or 3 numerical values ending with
+# a newline
+pattern = re.compile('([v]\d+\.)+(\d+\.){0,1}(\d)(?:$|\n)')
```

Comment:
In other places, we use [this regex](https://github.com/EDITD/docs/blob/master/scripts/termtools/release.sh#L24) - which doesn't need the `v` - which we don't have in some repos. What do you think about using the same regex and then you don't need the `[1:]` further down?